api/handlers.go

```diff
@@ -91,13 +91,17 @@ func makeTxRoute(router gin.IRouter, api blockatlas.Platform, path string) {
 
 		page := make(blockatlas.TxPage, 0)
 		for _, tx := range txs {
+			if tx.Direction != "" {
+				goto AddTx
```

Comment:
is it safe to use this? maybe consider if else instead?

Reply:
yess!!! `goto` method is normal and safe in Go, the golang compiler cannot compile the app if the `goto` skip handling some error or instantiate a variable. you can only use `goto` in safe places